packages/frontend/app/styles/components/user-guide-link.scss

```diff
@@ -0,0 +1,27 @@
+@use "../ilios-common/mixins" as cm;
+@use "../ilios-common/colors" as c;
+@use "../mixins" as m;
+
+.user-guide-link {
+  @include m.header-menu;
+
+  @include cm.for-tablet-and-up {
+    margin-left: 0.25rem;
+    margin-right: 0.5rem;
+  }
+
+  @include cm.for-phone-only {
+    /* stylelint-disable property-disallowed-list */
+    font-size: 3.5vw;
+  }
+
+  a {
+    svg {
+      align-items: center;
+      color: c.$white;
+      display: flex;
+      height: 1.85em;
```

Comment:
I'm wondering if this can be done as a percentage of the container. Not sure, just feels a little arbitrary and might break if we accidentally slightly change the header height.

Reply:
I changed this to use pixels instead. Using percentage of the container did not seem to work.